### PR TITLE
About page implementation

### LINKED ISSUE #75: Implement EDD Bookings About section
### Problem
Our customers should be able to learn more about product, suggest a feature or contact us. This actions will be available on about page.

### Solution
Implement EDD Bookings About section

### PR DESCRIPTION
**Pull request by romalytvynenko**
fixes #75

### CODE REVIEW

File: services.php [hunk 1]
```diff
@@ -126,12 +126,12 @@
...
         },
         /*
-         * The placeholder template for about and settings.
+         * The placeholder template for about page.
          *
          * @since [*next-version*]
          */
-        'eddbk_ui_coming_soon_template' => function (ContainerInterface $c) {
-            $templateFile = 'templates/about-settings-coming-soon.html';
+        'eddbk_ui_about_template' => function (ContainerInterface $c) {
```
Comment (XedinUnknown) [inline services.php]: How about something like [`_makeTemplate()`][`WpBookingsFrontUi#_makeTemplate()`] instead?

[`WpBookingsFrontUi#_makeTemplate()`]: https://github.com/RebelCode/rcmod-wp-bookings-front-ui/blob/develop/src/WpBookingsFrontUi.php#L242

File: src/WpBookingsUiModule.php [hunk 2]
```diff
@@ -497,15 +500,21 @@ function () use ($c) {
             }
         );
 
-        add_submenu_page(
+        $aboutContext = [
```
Comment (XedinUnknown) [inline src/WpBookingsUiModule.php]: I would prefer if the things that relate to the page itself would be in a separate method.

**Review by XedinUnknown — CHANGES_REQUESTED** [review]
Some small adjustments; otherwise good.

File: services.php (hunk 1 above)
Comment (romalytvynenko) [reply services.php]: Thought about this. The problem is that all services in this package are located in separate file, `services.php`, and services definition is the function that returns array. You sent a link on method that is a member of module class, and in that module all services are defined in the module class, so we have access to properties (`$this->templateFactory`).

So can we live this simple functions as is?

File: src/WpBookingsUiModule.php [hunk 3]
```diff
@@ -500,25 +501,38 @@ function () use ($c) {
...
+    /**
+     * Get context for about page.
+     *
+     * @since [*next-version*]
+     *
+     * @param array|stdClass|ContainerInterface $urls Map of urls for context.
+     *
+     * @return array Context for about page.
+     */
+    protected function _getAboutPageContext($urls)
```
Comment (XedinUnknown) [inline src/WpBookingsUiModule.php]: I was hoping you would just make a separate method that renders the page. The context can be created in that method. What's important is that the concerns are separated. Right now, there are still more reasons for `_adminMenu()` to change than there should be. This method is about the menu, and not about rendering a page.

**Review by XedinUnknown — APPROVED** [review]

**Pull request merged**